CONTRIBUTING.md

```diff
@@ -62,7 +62,8 @@ relates to existing content. Get approval to write your article.
 **Step 4:** Write your article, placing the article in its own folder and any
 needed images in a _static folder located in the same folder as the article.
 Be sure to follow the proper Markdown syntax. If you have code samples,
-place them in a folder within the `/samples/` folder.
+place them in a folder within the `/samples/` folder.  For writing guidelines see
+[our style guide](/styleguide.md)
```

Comment:
Should it be (re. punctuation)? "For writing guidelines, see [our style guide](/styleguide.md)."

Also, since we are reviewing this file, I think there are a couple of things that could be improved here:
1- We start the topic with the build process. This can be scary as the first thing you see to contribute. I imagine that not everybody contributing will need to build the docs. Am I wrong?
2- The guidelines are written from the standpoint of building new articles. What if I just want to edit a topic that I found an issue with? Do I also need to file an issue and get approval?